src/style.css

```diff
@@ -0,0 +1,87 @@
+@import url("https://fonts.googleapis.com/css?family=Roboto:400,500,700&display=swap");
+
+body {
+  margin: 0;
+}
+
+.card {
+  width: 200px;
+  font-family: "Roboto", sans-serif;
+  color: #000;
+  padding: 30px 14px;
+  font-size: 11px;
+  line-height: 13px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  box-sizing: border-box;
+  border-radius: 5px;
+}
+
+.card__img {
+  display: block;
+  width: 160px;
+  height: 134px;
```

Comment:
Try to use only height or width to best sizing.

Reply:
Done, thank you for review.